src/Component/AutoMapper/Extractor/MappingExtractor.php

```diff
@@ -88,6 +88,7 @@ public function getWriteMutator(string $source, string $target, string $property
         }
 
         if (PropertyWriteInfo::TYPE_ADDER_AND_REMOVER === $writeInfo->getType()) {
+            $type = WriteMutator::TYPE_ADDER;
```

Comment:
Why not using TYPE_METHOD ? Is there a big difference ?

Reply:
Because we need to know specifically if the method is an adder here : https://github.com/janephp/janephp/pull/553/files#diff-70128428cd4c89474bd81644c55cc91a88b44348ffec04444a1c5fe5056a3a59R149

Reply:
Ok, please make it more generic as Symfony PropertyInfo does "ADDER_REMOVER"